openrpc/src/methods/getHealth.json

```
@@ -15,6 +15,18 @@
                 "status": {
                     "type": "string",
                     "description": "\"healthy\""
+                },
+                "latestLedger": {
+                    "type": "number",
+                    "description": "Most recent known ledger sequence"
+                },
+                "oldestLedger": {
+                    "type": "number",
+                    "description": "Oldest ledger sequence kept in history"
+                },
+                "ledgerRetentionWindow": {
+                    "type": "number",
+                    "description": "Maximum retention window configured. When the window is full: ledgerRetentionWindow = latestLedger - oldestLedger"
```

```suggestion
                    "description": "Maximum retention window configured. A full window state can be determined via: ledgerRetentionWindow -1 <= latestLedger - oldestLedger"
```
this is based on observations from `getHealth` on rpc deployments with full window, the response values equate to at most `ledgerRetentionWindow - 1`

---

That's a bug then ...

---

Actually, it isn't. If the window is, say, 2, the first ledger could be 1 and the latest 2: `2 - 1 = 1` and not two